alpine-rails-buildkite/etc/sv/buildkite

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+export HOME=/opt/buildkite
+export GEM_HOME=${HOME}/.gem
+
+cd $HOME
```

Comment:
Yeah. I should not have pushed that early I planned to have a dedicated section we pop from into the root image. Sorry for leading you to that redundant flags.